core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -154,83 +159,55 @@ var ExplorationPlayerPage = function() {
   };
 
   this.reportExploration = async function() {
-    await waitFor.elementToBeClickable(
-      reportExplorationButton,
-      'Report Exploration Button takes too long to be clickable');
-    await reportExplorationButton.click();
+    await action.click('Report Exploration Button', reportExplorationButton);
     let radioButton = await element.all(by.tagName('input')).get(0);
     await waitFor.visibilityOf(
       radioButton, 'Radio Buttons takes too long to appear');
-    await radioButton.click();
+    await action.click('Radio Button', radioButton);
     let textArea = element(by.tagName('textarea'));
-    await textArea.sendKeys('Reporting this exploration');
+    await action.sendKeys('Text Area', textArea, 'Reporting this exploration');
     let submitButton = await element.all(by.tagName('button')).get(1);
-    await submitButton.click();
+    await action.click('Submit Button', submitButton);
     let afterSubmitText = await element(by.tagName('p')).getText();
     expect(afterSubmitText).toMatch(
       'Your report has been forwarded to the moderators for review.');
   };
 
   this.viewHint = async function() {
-    var until = protractor.ExpectedConditions;
-    const WAIT_FOR_FIRST_HINT_MSEC = 60000;
     // We need to wait some time for the solution to activate.
-    await browser.wait(
-      until.elementToBeClickable(viewHintButton), WAIT_FOR_FIRST_HINT_MSEC,
-      '"View Hint" button takes too long to be clickable');
-    await viewHintButton.click();
+    await action.click('View Hint Button', viewHintButton);
     await clickGotItButton();
   };
 
   this.viewSolution = async function() {
-    var until = protractor.ExpectedConditions;
-    const WAIT_FOR_SUBSEQUENT_HINTS = 30000;
     // We need to wait some time for the solution to activate.
-    await browser.wait(
-      until.elementToBeClickable(viewSolutionButton), WAIT_FOR_SUBSEQUENT_HINTS,
-      '"View Solution" button takes too long to be clickable');
```

Comment:
This is required, please add it back. We specify a custom wait time in this case.